react-basic-practice/src/components/Button/index.tsx

```diff
@@ -5,11 +5,10 @@ interface CustomButtonProps extends ButtonProps {
   label: string;
 }
 
-const Button: React.FC<CustomButtonProps> = ({ label, variant, size, ...rest }) => {
-  const styles = useStyleConfig("Button", { variant, size });
```

Comment:
Pls remove redundant code. Verify for all

Reply:
i done remove it